samples/elsa/Core_7/Messages/Messages.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus" Version="7.6.0" />
+    <PackageReference Include="NServiceBus" Version="[7.6.0, 8.0.0)" />
```

Comment:
Why isn't this 7.7.4?

Reply:
Think I manually undid it, but kep the range version. I'll update to `7.7.0`. We can decide to use the bracket format e.g. `[7.7.0]` in a separate PR (as we need to do it consistently)

Reply:
FYI we never used the bracket format in docs so far

Reply:
Same as above.